src/event/special_user_message.rs

```diff
@@ -358,6 +372,36 @@ fn deploy_command<'a>(
             .send()
             .await
             .map_err(|e| format!("API Error: {:?}", e))?;
+        let res_json = res
+            .json::<BuildsResponse>()
+            .await
+            .map_err(|_| "Invalid API Response")?;
+        banker::message(format!("Build submitted! URL: {}", res_json.setup_log)).await?;
+        Ok(())
+    }
+    .boxed()
+}
+
+lazy_static::lazy_static! {
+    pub static ref RESTART_SERVER: SpecialUserMessageTrigger = SpecialUserMessageTrigger {
+            regex: Regex::new("<@([A-z|0-9]+)> goblin chant").unwrap(),
```

Comment:
wait, a second trigger on goblin chant? I'm wondering if the dispatching code that applies these events will work with that

Reply:
(or if this was even intentional? did you mean for a different keyword?)

Reply:
yeah it was supposed to be `restart`, must have missed that

Reply:
yay!!! review working as intended :tada: